templates/CRM/Contribute/Form/UpdateSubscription.tpl

```diff
@@ -52,7 +52,9 @@
     {/if}
   </table>
 
-  {include file="CRM/common/customDataBlock.tpl"}
+  {if !$self_service}
```

Comment:
Can we call this isShowCustomData - I prefer variables that say what they mean rather than having implicit meanings

Reply:
hmm well this variable also use ised to hide other back office stuff see L48 of this file where it is used to hide the nofity, campaign and financial_type fields so i think it is appropriate name given the context